apps/ledger-live-desktop/src/renderer/modals/Platform/Exchange/CompleteExchange/Body.tsx

```diff
@@ -30,6 +30,13 @@ export type Data = {
   amountExpectedTo?: number;
 };
 
+export function isCompleteExchangeData(data: unknown): data is Data {
+  if (data === null || typeof data !== "object") {
+    return false;
+  }
+  return "signature" in data && "binaryPayload" in data ? true : false;
```

Comment:
```suggestion
return "signature" in data && "binaryPayload" in data;
```

It should work without checking if true is true?

Reply:
you're right 🤦